lti_consumer/lti_1p3/constants.py

```diff
@@ -52,6 +52,9 @@
 ]
 
 
+LTI_DEEP_LINKING_ACCEPTED_TYPES = []
```

Comment:
@giovannicimolin, I think you've missed adding ``ltiResourceLink`` in ``LTI_DEEP_LINKING_ACCEPTED_TYPES``. Deep Linking Launch fails without that.

Reply:
@shimulch This was by design, the main PR has the required change, but adding it to this list meant that I would need to add the serializers and support for saving `ltiResourceLink` here.

The changes you are looking for are on https://github.com/edx/xblock-lti-consumer/pull/127, where I implement the models and the `ltiResourceLink` support.

Reply:
@giovannicimolin, I think that might be ok. As the whole reason for separate PRs is just to ease the review process and all three need to get merged for Deep linking to work anyway. It would've been nice if these PRs were self-sufficient without dependency, but that might make the splitting process really hard and time-consuming. 

You should add this to the test instruction, maybe on the patch section (step 6).